src/observable/throw.ts

```diff
@@ -4,7 +4,7 @@ import {Subscription} from '../Subscription';
 
 export class ErrorObservable<T> extends Observable<T> {
 
-  static create<T>(error: T, scheduler?: Scheduler) {
+  static create<T>(error: any, scheduler?: Scheduler) {
```

Comment:
Would you able to elaborate some snippets represents those cases?
In my understanding, current `<T>` in function signature won't be used all time for example

`Observable.throw('xxx');` 

since error object is explicitly specified to `any`, `T` won't be specified. It's not type of `Observable` class itself in `ErrorObservable<T>`, since function itself redefines generic for function only.

Simplifying current PR will looks as

``` 
class A<T> {
 doSomething<R>(value: any);
}
```

where `<R>` is not being used anywhere. I could possibly miss something, it'd be appreciate if you could bring up snippets those type's being consumed.